jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/asserts/TimePickerAssert.java

```diff
@@ -8,216 +8,161 @@
+            Arrays.asList(expectedEnabledNumbers), actualEnabledNumbers);
+        jdiAssert(actualEnabledNumbers, Matchers.contains(expectedEnabledNumbers), errorMsg);
         return this;
     }
 
-    @Override
-    @JDIAction("Assert that '{name}' field's color is correct")
-    public TimePickerAssert backgroundColor(String color) {
-        String actualFieldBackgroundColor = element().fieldBackgroundColor();
-        jdiAssert(actualFieldBackgroundColor, Matchers.is(color), String.format("Actual field background color '%s' " +
-                "is not equal to expected '%s'", actualFieldBackgroundColor, color));
+    @JDIAction("Assert that '{name}' has '{0}' selected number")
+    public TimePickerAssert selectedNumber(int expectedSelectedNumber) {
+        int actualSelectedNumber = element().selectedNumber();
+        String errorMsg = format("Expecting '%s' numbers to be selected, But selected number was '%s'",
+            expectedSelectedNumber, actualSelectedNumber);
+        jdiAssert(actualSelectedNumber, Matchers.is(expectedSelectedNumber), errorMsg);
         return this;
     }
 
-    @JDIAction("Assert that '{name}' title field has correct hours")
-    public TimePickerAssert hours(String hours) {
-        String actualHours = element().getHours();
-        jdiAssert(actualHours, Matchers.is(hours), String.format("Actual hours '%s' are not equal to expected '%s'",
-                actualHours, hours));
-        return this;
+    @JDIAction("Assert that '{name}' has disabled numbers from {0} to {1} inclusive")
+    public TimePickerAssert disabledNumbersRange(int from, int to) {
+        return disabledNumbers(IntStream.rangeClosed(from, to).boxed().toArray(Integer[]::new));
     }
 
-    @JDIAction("Assert that '{name}' title field has correct minutes")
-    public TimePickerAssert minutes(String minutes) {
-        String actualMinutes = element().getMinutes();
-        jdiAssert(actualMinutes, Matchers.is(minutes), String.format("Actual minutes '%s' are not equal to " +
-                "expected '%s'", actualMinutes, minutes));
-        return this;
+    @JDIAction("Assert that '{name}' has enabled numbers from {0} to {1} inclusive")
+    public TimePickerAssert enabledNumbersRange(int from, int to) {
+        return enabledNumbers(IntStream.rangeClosed(from, to).boxed().toArray(Integer[]::new));
     }
 
-    @JDIAction("Assert that '{name}' title field has correct seconds")
-    public TimePickerAssert seconds(String seconds) {
-        String actualSeconds = element().getSeconds();
-        jdiAssert(actualSeconds, Matchers.is(seconds), String.format("Actual seconds '%s' are not equal to " +
-                "expected '%s'", actualSeconds, seconds));
+    @JDIAction("Assert that '{name}' is 12h")
+    public TimePickerAssert format12() {
+        String errorMsg = "Timepicker have 24h format";
+        jdiAssert(element().isAmPm() ? "12h" : "24h", Matchers.is("12h"), errorMsg);
```

Comment:
The value must be compared with true or false as we already have custom error message

Reply:
Done this according to the:
[Pages and elements management. Not-allowed](https://jdi-family.atlassian.net/wiki/spaces/JDIL/pages/47349767/Pages+and+elements+management#Not-allowed)
> Do not use [Matchers.is](http://matchers.is/)(true/false) in the asserts. A ternary operator should be used instead.